src/main/java/org/sciborgs1155/robot/Intake/Intake.java

```diff
@@ -0,0 +1,49 @@
+package org.sciborgs1155.robot.Intake;
+
+import static org.sciborgs1155.robot.Intake.IntakeConstants.*;
+
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import monologue.Logged;
+import monologue.Monologue.LogBoth;
+import org.sciborgs1155.robot.Robot;
+
+public class Intake extends SubsystemBase implements Logged {
+
+  final PIDController pid = new PIDController(kp, kd, ki);
+
+  public final IntakeIO intake = Robot.isReal() ? new RealIntake() : new SimIntake();
+  @LogBoth public double targetSpeed = pid.getSetpoint();
+
+  @LogBoth
+  public double speed() {
+    return intake.getSpeed();
+  }
+
+  @Override
+  public void periodic() {
+    intake.setVoltage(pid.calculate(intake.getSpeed(), targetSpeed));
+  }
+
+  public Command intake() {
+    return runOnce(
+        () -> {
+          pid.setSetpoint(INTAKE_SPEED);
+        });
+  }
+
+  public Command stop() {
```

Comment:
this (and hopper too) is a situation where you can just set the default command inside the constructor if you want, rather than in Robot.java. Something like drive is different because the default command uses the controllers, which are in Robot.java